eclipse-plugin/service/cc.codechecker.service.thrift/src/main/java/cc/codechecker/api/thrift/bug/path/BugPathActionThriftImpl.java

```diff
@@ -72,8 +72,7 @@ protected String getProtocolUrlEnd(ProblemInfoRequest request) {
             System.out.println("Event: " + bpe.getStartLine() + " - " + bpe.getStartCol() + " - "
                     + bpe.getMsg());
         }
-
-        Collections.sort(listBuilder);
+        //Collections.sort(listBuilder);
```

Comment:
You can just delete this line, the original will be in the VCS history if needed.